crates/viewer/re_renderer/src/context_test.rs

```diff
@@ -1,20 +1,30 @@
 //! Extensions for the [`RenderContext`] for testing.
 //!
+use std::sync::LazyLock;
+
 use crate::{RenderConfig, RenderContext, device_caps};
 
+/// Adapter shared with all tests.
+///
+/// Devices are not re-used since we want to isolate wgpu resources.
+static TEST_ADAPTER: LazyLock<wgpu::Adapter> = LazyLock::new(|| {
```

Comment:
could go into a separate PR. But on my Windows box this turned out necessary for executing the new tests - there seems to be some race condition in wgpu (or my driver?) when spamming creation of new adapters in parallel. (also, it's slow ;))
Worth deeper investigation from a wgpu pov, but need to isolate it first